mrp_production_picking_type_from_route/models/mrp_production.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    # Skip picking_type_id as a onchange trigger, even though the
+    # extended method is an onchange for picking_type_id too.
+    # If triggered also from picking_type_id changes, it would be impossible
+    # to adjust picking_type_id manually.
+    @api.onchange("product_id", "company_id")
```

Comment:
```suggestion
    @api.onchange("product_id", "move_raw_ids", "company_id")
```
keep the move raw ids in the onchange decorator from the original method https://github.com/odoo/odoo/blob/e59261506f9258112475c7a7798cb2fd15e39b5b/addons/mrp/models/mrp_production.py#L816